### PR TITLE
Add LugasaCrystal's pin to the map

### PR DESCRIPTION
**Pull request by LugasaCrystal**
Add LugasaCrystal to the map
first issue

### CODE REVIEW

File: _pins/LugasaCrystal.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandel:LugasaCrystal
```
Comment (crichID) [inline _pins/LugasaCrystal.json]: @LugasaCrystal you will need to fix the spelling of `githubHandle` and add a space after the `:` so the tests will pass.

File: _pins/LugasaCrystal.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+Latitude: 0.347596
```
Comment (crichID) [inline _pins/LugasaCrystal.json]: Latitude and Longitude should both start with lowercase l's

**Comment by ndfuller** [comment]
@LugasaCrystal update spelling of handle. looks great

**Comment by brianamarie** [comment]
@LugasaCrystal  I went ahead and fixed the syntax issues in [571704f](https://github.com/githubschool/open-enrollment-classes-introduction-to-github/pull/5663/commits/e122d8f18d41b83af6fd617dd5ba37a017924980). Please take a quick look at the changes. Once the checks have finished, please feel free to click the big green **Merge pull request** button. If you haven't merged it by April 28, I will go ahead and do it just to keep the repository clean!

**Pull request merged**